src/qibotn/backends/__init__.py

```diff
@@ -26,7 +26,11 @@ def load(platform: str, runcard: dict = None) -> QibotnBackend:
         if platform == "cutensornet":  # pragma: no cover
             return CuTensorNet(runcard)
         elif platform == "quimb":  # pragma: no cover
-            return QuimbBackend(runcard)
+            quimb_backend = kwargs.get("quimb_backend", "numpy")
+            contraction_optimizer = kwargs.get("contraction_optimizer", "auto-hq")
+            return QuimbBackend(
+                quimb_backend=quimb_backend, contraction_optimizer=contraction_optimizer
+            )
```

Comment:
Another way of doing this is to define a constant dictionary of defaults options, and update it with the `kwargs`.

I.e.
```py
QUIMB_DEFAULTS = {
    "quimb_backend": "numpy",
    "contraction_optimizer": "auto-hq",
}
```
and

```suggestion
            return QuimbBackend(**(QUIMB_DEFAULTS | kwargs))
```